db/schema.rb

```diff
@@ -517,18 +517,44 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "test_group_results", id: :serial, force: :cascade do |t|
+    t.integer "test_group_id"
```

Comment:
While we're changing the schema, it would be great to make this non-null.

Reply:
It should be nullable, for the case when the test group (previously script) gets deleted while tests are being done on the autotester. It actually really happened in production.